### ISSUE TITLE
[Issue]: Can't save zip file to a different drive

### ISSUE THREAD

**Issue by cagure** [MichaelYochpaz/iSubRip#44]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing open issue for the issue I want to report.

### OS Type

Windows

### Python Version

3.8.10

### Package Version

2.4.1

### Description
Hi

if i use `folder = "C:\\Users\\Downloads\\Itunes\\"` everything works fine,  
but if I change the disk to `folder = "E:\\iTunes\\"`, it doesn't let me download the subtitles with the zip.
but I can download them in any format.
with the old version there wasn't this problem.  

Thanks

### Config

```TOML
folder = "E:\\iTunes\\"
```

### Output Log
```
[WinError 17] The system cannot move the file to a different disk drive
```

**Comment by MichaelYochpaz** [comment]
Hey, thanks for reporting.  
Seems to be the same issue as https://github.com/MichaelYochpaz/iSubRip/issues/41, I just forgot to update the code for the zip file as well on the last release (2.4.1).

I'll release an updated version with a fix for that soon.

**Comment by cagure** [comment]
> I'll release an updated version with a fix for that soon.

Thanks

**Comment by MichaelYochpaz** [comment]
Fixed on the new 2.4.2 version
